README.md

```diff
@@ -121,23 +121,23 @@ Restart Docker:
 sudo service docker restart
 ```
 
-### Building the images
+### Building and running the images
 
 :warning: Do not unzip the `1024.zip` file.
 
-Build the Docker images as follows.
-
-
-```bash
-./docker.sh -b
-```
-
-Once they are built, set the desired configuration in `config/config.json`.
-
-Then, run them using:
-
-```bash
-./docker TODO
-```
-
-TODO: set num threads
+- `git pull`
+- Save measurement results (`output/*`) if necessary
+- Clean all files to have a clean build:\
```

Comment:
Do we really need this? With a better `.dockerignore` we shouldn't have to do this.

Reply:
I believe @marci543 compiled the current `.dockerignore` so that it is a superset of the `.gitignore` file. So maybe this instruction is indeed superfluous.

Reply:
Currently it is needed.
E.g. during the build-time `config/config-docker-*.json` files are copied to the images as a default configuration and it defines the tools to build and used when the image is run stand-alone, not from `docker.sh`.

The `./docker.sh -r` command mounts the same config files from the host to the container to allow the user to change the config without a rebuild.

Since the same file is used for "permanent" build-time steps and "temporary" run-time steps, we have to clean the files to avoid changing the image.

Reply:
>I believe @marci543 compiled the current .dockerignore so that it is a superset of the .gitignore file.

That was the idea, but I haven't collected all of them, only a few.